src/server/scripts/Northrend/Ulduar/Ulduar/instance_ulduar.cpp

```diff
@@ -1039,14 +1042,8 @@ class instance_ulduar : public InstanceMapScript
                             for (auto const& vehicleGuid : LeviathanVehicleGUIDs)
                             {
                                 if (Creature* vehicleCreature = instance->GetCreature(vehicleGuid))
-                                {
-                                    if (Vehicle* vehicle = vehicleCreature->GetVehicleKit())
-                                    {
-                                        vehicle->RemoveAllPassengers();
-                                        vehicleCreature->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
-                                        vehicleCreature->DespawnOrUnsummon(5 * MINUTE * IN_MILLISECONDS);
-                                    }
-                                }
+                                    DespawnLeviatanVehicle(vehicleCreature);                                   
+                                
```

Comment:
Looks like you left in redundant whitespace. Just remove the blank line.

Reply:
Better...
can be changed by:
```cpp
for (auto const& vehicleGuid : LeviathanVehicleGUIDs)
    if (Creature* vehicleCreature = instance->GetCreature(vehicleGuid))
        DespawnLeviatanVehicle(vehicleCreature);
```

And has white space after ```DespawnLeviatanVehicle(vehicleCreature);``` remove please.

Reply:
After this last change i will merge 💃

Reply:
Done and commited!

Reply:
Sorry for my brackets, white spaces and tabs, I am new committing to TC